server/db/seed.js

```diff
@@ -58,27 +58,25 @@ const scriptParams = {
 const fullContactFirstName = faker.name.firstName();
 const fullContactLastName = faker.name.lastName();
 const fullContactEmail = `${fullContactFirstName}.${fullContactLastName}@notrealemail.com`;
+
 const contactParams = [
   {
-    first_name: faker.name.firstName(),
-    phone_number: `555-555-${getRandomFourDigitInt()}`
-  },
-  {
-    first_name: faker.name.firstName(),
-    phone_number: `555-555-${getRandomFourDigitInt()}`
-  },
-  {
-    first_name: faker.name.firstName(),
-    phone_number: `555-555-${getRandomFourDigitInt()}`
-  },
-  {
     first_name: fullContactFirstName,
     last_name: fullContactLastName,
     phone_number: `555-555-${getRandomFourDigitInt()}`,
     email: fullContactEmail,
     external_id: 'external_id'
   }
 ];
+
```

Comment:
Have you considered:
```
let contacts = new Array(45).map(() => {
  first_name: faker.name.firstName(),
  phone_number: '555-555-${getRandomFourDigitInt()}'
});
```

Reply:
I knew there had to be a sleeker way. Was going for speed based on what I knew - I'll work to update!

Reply:
@perfectlynormalbeast I was able to do this but had to make the empty array fields null in order to map to them. Then, the linter was giving me trouble with the phone_number template strings, so I separated that out. Let me know what you think!